tests/GenericSparqlTest.php

```diff
@@ -491,7 +491,10 @@ public function testQueryTypes()
       'http://www.skosmos.skos/test-meta/TestClass' => array(
         'superclass' => 'http://www.w3.org/2004/02/skos/core#Concept',
         'label' => 'Test class'
-      )
+      ),
+      'http://www.w3.org/2004/02/skos/core#Collection' => array(),
+      'http://purl.org/iso25964/skos-thes#ThesaurusArray' => array(),
+      'http://purl.org/iso25964/skos-thes#ConceptGroup' => array()
```

Comment:
Had a look at [`testconfig.ttl`](https://github.com/NatLibFi/Skosmos/blob/master/tests/testconfig.ttl), and I think the function shuold return these new types now as they are types used in vocabularies there. These extra values are added now since the query is searching types in all graphs for all configured vocabs.